libs/blocks/article-feed/article-helpers.js

```diff
@@ -81,12 +81,18 @@ function loadArticleTaxonomy(article) {
   // for now, we can only compute the category
   const { tags, path } = clonedArticle;
 
+  let topics;
+
   if (tags) {
-    const topics = tags
-      .replace(/[["\]]/gm, '')
-      .split(',')
-      .map((t) => t.trim())
-      .filter((t) => t && t !== '');
+    if (Array.isArray(tags)) {
+      topics = tags.map((t) => t.trim()).filter((t) => t && t !== '');
```

Comment:
Nit: 
- can remove the round parentheses around `t`;
- no need to check that `t !== ''`, the empty string is falsy and will be filtered out anyway;
```suggestion
      topics = tags.map(t => t.trim()).filter(t => t);
```

Reply:
Will leave as is. This code was pre-existing and I'm trying to keep my changes to a minimum.